.github/workflows/run-tests.yml

```
@@ -0,0 +1,24 @@
+---
+name: Run tests
+
+on: [push]
```

Let's also run for PRs, and `workflow_dispatch` adds a button to the GitHub UI to be able to trigger new builds.

```suggestion
on: [push, pull_request, workflow_dispatch]
```

---

Thanks!